### ISSUE TITLE
windows blocks the payload

### ISSUE THREAD

**Issue by Parshuram2** [stormshadow07/HackTheWorld#15]
Windows automatically blocks the file. One has to manually unblock th application. Is this to be expected?

**Comment by GetRektBoy724** [comment]
yes, of course, this is expected. You can not expect a forever FUD payload that is publicly accessible lol, the Blue Team is not stupid ;)
